crates/eframe/src/native/run.rs

```diff
@@ -22,7 +22,17 @@ pub use epi::NativeOptions;
 #[derive(Debug)]
 enum EventResult {
     Wait,
+    /// Causes a synchronous repaint inside the event handler. This should only
+    /// be used in special situations if the window must be repainted while
+    /// handling a specific event. This occurs on Windows when handling resizes.
+    ///
+    /// `RepaintAsap` introduces a one-frame delay, and should therefore only be
```

Comment:
Isn't that sentence contradicts the intention of this variant? If you paint immediately then where one-frame delay can get from? From my understanding of the situation, the delay would be in `RequestNext`, no?

Reply:
By one-frame delay I mean `RepaintNow` (I renamed it) blocks the event loop for a frame. It delays further processing of events until the frame has been synchronously produced. This is normal and expected when it comes to resize events on Windows, since the compositor uses them to tell when the application has handled the resize. But as far as I can tell, no other event acts this way. (I used `RepaintNow` for `Resumed` events as well, but in hindsight that is probably not necessary.)

Reply:
Ah.. Then maybe rewrite this sentence, because it is a very unclear, that the time of current frame is just increased (so it takes a time that normally two frames take)

Reply:
> it is a very unclear, that the time of current frame is just increased (so it takes a time that normally two frames take)

This doesn't increase the time of the current frame. This creates a brand new frame into the current iteration of the event loop.